src/saving/SaveManagerGUI.tscn

```diff
@@ -142,9 +143,22 @@ size_flags_vertical = 5
 popup_exclusive = true
 window_title = "Delete selected Save(s)?"
 dialog_text = "Text not set..."
+
+[node name="DeleteOldConfirmDialog" type="ConfirmationDialog" parent="CenterContainer/VBoxContainer"]
+margin_right = 184.0
+margin_bottom = 97.0
+rect_min_size = Vector2( 0, 0 )
+size_flags_horizontal = 5
+size_flags_vertical = 5
+popup_exclusive = true
+window_title = "Delete old Saves"
```

Comment:
This should probably end in a question mark as the popup is a confirmation on deleting them.